packs/data/spells.db/undeaths-blessing.json

```
@@ -26,7 +27,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You instill within a creature the touch of the grave. For the duration, <em>@Compendium[pf2e.spells-srd.Harm]{Harm}</em> spells treat the creature as undead and <em>@Compendium[pf2e.spells-srd.Heal]{Heal}</em> spells treat the creature as living. In addition, harm spells gain a +2 status bonus to the Hit Points restored to the target.</p>\n<p>If the target wishes to avoid the spell, it can attempt a Will save to reduce the effects.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target heals half as much from <em>heal</em> and takes half as much damage from <em>harm</em> for 1 round.</p>\n<p><strong>Failure</strong> Effects as described in the text.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The status bonus to the Hit Points restored increases by 2.</p>"
+            "value": "<p>You instill within a creature the touch of the grave. For the duration, @Compendium[pf2e.spells-srd.Harm]{Harm} and @Compendium[pf2e.spells-srd.Heal]{Heal} spells treat the creature as undead. In addition, harm spells gain a +2 status bonus to the Hit Points restored to the target.</p>\n<p>If the target wishes to avoid the spell, it can attempt a Will save to reduce the effects.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target heals half as much from <em>heal</em> and takes half as much damage from <em>harm</em> for 1 round.</p>\n<p><strong>Failure</strong> Effects as described in the text.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The status bonus to the Hit Points restored increases by 2.</p>"
```

```suggestion
            "value": "<p>You instill within a creature the touch of the grave. For the duration, <em>@Compendium[pf2e.spells-srd.Harm]{Harm}</em> and <em>@Compendium[pf2e.spells-srd.Heal]{Heal}</em> spells treat the creature as undead. In addition, harm spells gain a +2 status bonus to the Hit Points restored to the target.</p>\n<p>If the target wishes to avoid the spell, it can attempt a Will save to reduce the effects.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target heals half as much from <em>heal</em> and takes half as much damage from <em>harm</em> for 1 round.</p>\n<p><strong>Failure</strong> Effects as described in the text.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The status bonus to the Hit Points restored increases by 2.</p>"
```
We italicize even the links to match Paizo style for referring to spell names.